README.md

```diff
@@ -26,12 +37,42 @@ Current research is targeted to explore and analyze data from WorldSkills Russia
 - Whether repeated participation in competitions significantly improves a competitor's average score?
 - Whether repeated participation of a compatriot expert significantly improves his/her compatriot competitors' average results?
 
+These questions are formulated as followed:
+
+1. Impact of repeated participation on competitor's average score.
+
+_null hypothesis (h0):  
+The repeated participation in WorldSkills competitions does not significantly improve a competitor's average score._   
+
+_alternative hypothesis (h1):  
+The repeated participation in WorldSkills competitions significantly improves a competitor's average score._
+
+2. Effect of repeated participation of a compatriot expert on competitor results
+
+_null hypothesis (h0):  
+The repeated participation of a compatriot expert does not significantly improve his/her competitor's (average) results in the competitions of WorldSkills Russia._  
+
+_alternative hypothesis (h1):  
+The repeated participation of a compatriot expert significantly improves his/her compatriot competitor's average results in the competitions of WorldSkills Russia._
+
 ## Operationalization of the research questions
 <!-- Describe the data that will be used and how the questions will be answered on the basis of this data. The data analysis itself is not yet described here. So do tell ‘Data file [X] comes from [Y] and can answer the posed questions because [Z]’, but do not yet describe the data itself -->
 
 Each competition event uses the Competition Information System (CIS) to record competition results. A separate instance of CIS is used for each event, so all event results are stored in a separate database instance. In order to have an overview of all competition events, WorldSkills Russia has developed its own aggregation system called Electronic System for Internet Monitoring (eSIM).
 
-The raw data used for this research was kindly provided by the autonomous non-profit organisation "Agency for the Development of Professional Excellence (WorldSkills Russia)" and is essentially a dump from the eSIM database, specifically from the competition results table. Therefore, the observation unit is a skills competition result for a given competitor for a given competition event.
+The raw data used for this research was kindly provided by the autonomous non-profit organization "Agency for the Development of Professional Excellence (WorldSkills Russia)" and is essentially a dump from the eSIM database, specifically from the competition results table. Therefore, the observation unit is a skills competition result for a given competitor for a given competition event.
+
+The following steps will make sure to operationalize the data so that the research questions that are formulated will be answered:
+- Data preparation  
+- Data cleaning
+- Calculate initial summary statistics  
```

Comment:
Do we understand what this summary statistics will represent in the context of the current research domain?

Reply:
Not yet, I added this part when looking back in the book for which steps to take early on. I will check the summary statistics in the script and let you know if this will be of any help to the research questions. I'll edit it so we know that it is pending.